open-metadata-implementation/user-interfaces/ui-admin-services/README.md

```diff
@@ -12,46 +12,64 @@ on the OMAG UI platform and querying the runtime (operational) state of these ui
 
 An example rest call to configure the UI is a post call to https://localhost:8443/open-metadata/ui-admin-services/users/<userid>/servers/<server>/configuration
 with body:
-
-{
+```
+{ 
    "class":"org.odpi.openmetadata.userinterface.adminservices.configuration.properties.UIServerConfig",
-   "localServerName":"ccc",
+   "localServerId":"90083805-8ecf-45a6-9d7e-e0895592c090",
+   "localServerName":"aaa",
```

Comment:
think of a good server name?

Reply:
How about just changing to something like 'myuiserver' or 'myserver' . this isn't coco pharma specific that we need as it's generic doc. Just looks slightly better than aaa?

Reply:
changed to cocoUIS1 - which seems to follow the existing server naming pattern